### PR TITLE
Improve Docs

### PR DESCRIPTION
**Pull request by utnaf**
This PR is for creating the final doc pages.

http://5.9.211.195:8000/neo4j-spark-docs/1.0.0/

### CODE REVIEW

File: doc/docs/modules/ROOT/pages/reading.adoc [hunk 1]
```diff
@@ -0,0 +1,475 @@
...
+val spark = SparkSession.builder().getOrCreate()
+
+spark.read.format("org.neo4j.spark.DataSource")
+  .option("url", "bolt://localhost:7687")
+  .option("labels", "Person:Customer:Confirmed")
+  .load()
+----
+
+[NOTE]
+Label list can be speficied both with starting color or without it:
```
Comment (darabos) [inline doc/docs/modules/ROOT/pages/reading.adoc]:
```suggestion
Label list can be specified both with starting colon or without it:
```

(Sorry about the drive-by comment. I just looked at the main README and it looked outdated talking about Neo4j 3 and Spark 2 but then I spotted this PR and I'm very happy. Thanks for the work on the 4.0 Connector!)

File: src/test/scala/org/neo4j/spark/DataSourceReaderTSE.scala [hunk 2]
```diff
@@ -26,6 +26,13 @@ class DataSourceReaderTSE extends SparkConnectorScalaBaseTSE {
     assertTrue(df.select("<id>").collectAsList().get(0).getLong(0) > -1)
   }
 
+  @Test
+  def teestteest(): Unit = {
```
Comment (conker84) [inline src/test/scala/org/neo4j/spark/DataSourceReaderTSE.scala]: can we remove this?

**Review by conker84 — APPROVED** [review]
just a small thing otherwise looks good to me

**Comment by conker84** [comment]
thank you @utnaf @moxious
